### PR TITLE
Single window

### PR DESCRIPTION
**Pull request by JerPk**

### CODE REVIEW

File: src/main/java/level/Level.java [hunk 1]
```diff
@@ -66,11 +81,85 @@ public Level(int number) {
...
+
 		return tempBarriers;
 	}
+
+	/**
+	 * creates the Transition screen between levels.
+	 * 
+	 * @return panel the transition panel
+	 */
+	public JPanel createTransitionPanel() {
```
Comment (Mari3) [inline src/main/java/level/Level.java]: This method is way too long. Maybe try to break it up somehow.

**Comment by harveyvanveltom** [comment]
after the current flaws are fixed it is ready to be merged.

**Comment by JerPk** [comment]
I've merged the master into it, shortened the createTransitionPanel method and did some checking of the tools.

**Comment by Mari3** [comment]
Ok, can be merged now

**Pull request merged**